ush/templates/FV3LAM_wflow.xml

```diff
@@ -89,7 +89,7 @@ tasks; and the "FCST" type is used for the RUN_FCST_TN task.
 
 ]>
 
-<workflow realtime="T" scheduler="&SCHED;" cyclethrottle="3" cyclelifespan="01:00:00:00">
+<workflow realtime="T" scheduler="&SCHED;" cyclethrottle="15" cyclelifespan="01:00:00:00">
```

Comment:
I increased this to 25 in my dev2 PR so we don't have to mess with it again until we go sub-hourly.  Don't have to, just a suggestion